### ISSUE TITLE
Ridge Racer 7 (BLUS30001) Crashing in Main menu and General Desyncing

### ISSUE THREAD

**Issue by DrDrPepper** [RPCS3/rpcs3#9380]
## Quick summary
Please briefly describe what is not working correctly.
When you first boot the game and go past the the loading minigame and whatnot, every other time you "press start" on the title screen the game crashes. The other times the game progresses through the title screen fine but once you go to the  "ridge state grand prix" the game crashes there too. The other gamemodes on the main menu do not have any crashing bugs to speak of.

## Details
Every time the game crashes in said areas all have the same error code of RSX: nv406e::semaphore_acquire has timed out. semaphore_address= 0x40300800 it also deadlocks the Emulator whenever this happens. -> Picture in question 
![RPCS3 Error code](https://user-images.githubusercontent.com/75584729/101291039-2f7fff00-37c3-11eb-9081-2cc3d8ef437f.png) it also puts various other error codes that would most likely be in the Log but most of the time they do nothing to the normal gameplay. 

This is when the Graphics of the game starts to Desync, the audio and everything else about the game still go on as normal but the game freezes on one frame. I've even went back on the main menu on one attempt when the game was still frozen which was... interesting to say the least. 

The weirder part of this is too is that there is no rhyme or reason to it happening, I can't reliably reproduce the desyncing bug if I wanted to. And Contrary to what the People of the #help section of the RPCS3 discord say, increasing the driver wake up delay does not do anything to the game's playability, in fact in some cases it worsened it. Setting the driver wakeup delay to more than 80us will cause the game to just be a black screen on boot and it would not progress further.

The only thing I could think of that remotely helped with the playability of the game was turning on Accurate RSX reservation access and Setting the renderer to OpenGL, which stops some of the desyncs that happen.

#### 1. Please attach RPCS3's log.
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/5649427/RPCS3.log)

#### 3. Please attach screenshots of your problem.
![RPCS3 Crashing Main Menu](https://user-images.githubusercontent.com/75584729/101291647-a3bca180-37c7-11eb-8067-e4c7abb29ce0.png)
![RPCS3 Crashing Main Menu 2](https://user-images.githubusercontent.com/75584729/101291657-c2bb3380-37c7-11eb-9ba9-59e0dd2a3e2d.PNG)

#### 4. Please provide comparison with real PS3.
I do not have a real PS3 on me unfortunately
#### 5. Please provide your system configuration:
* OS Windows 10 Version 2004
* CPU AMD Ryzen 1500X OC'd to 3.8GHZ
* GPU Nvidia GTX 1050Ti
* Driver version 456.55
* etc.

#### Please include.
* Anything else you deem to be important
Also i've noticed that when i updated to PR #9337 the game was running at around 35 to 47 fps rather than the consistent 60fps with 1080p upscaling that i was using. just a minor thing though

**Comment by Pouncha** [comment]
BCES00009 works perfectelly in 1440p.
I just put Legacy shader mode for right graphics in Sunset Highs Race
Windows 10 1909 with I9 9900KF, GTX 1080 and drivers 457.51
Try to delete shaders in cache folder and HD1 folder then restart

**Comment by DrDrPepper** [comment]
Yeah i tried those things and the issues are still there, no big surprise lol. maybe i just have the cursed copy

**Comment by Augusto7743** [comment]
Ubuntu 20.10 FX-6300-4 GB-GT 640 having the same error message
"RSX: nv406e::semaphore_acquire has timed out. semaphore_address"
Here crashing some games even having more of 1,2 GB RAM free using SPU LLVM mode safe, mega or giga and enabling or disabling RSX reservation.

**Comment by Back2Life888** [comment]
Not only does it crashes in menus. It crashes on gameplay too.. Once you make it through gameplay, it will crash for some time. Its a problem for Vulkan. In Vulkan, the game crashes 2-4 times on menus and after the 2-4 times it crashes, you could now navigate on the menu. But it will also crash when you navigate to the menus either the game will crash or the screen will freeze. And of course, it happens on gameplay alot. If you are in OpenGL, well everything is stable with a slow performance and broken shaders. And don't worry, it crashes less. If it crashes once you press the start button, it won't crash again or it will crash again for the final time. The crashes in the rest of the menus are now rare to happen so if you are stuck in RSGP 13 because of the crashes, use OpenGL. There is another thing that if you navigate really fast on the Special Machines, you're game will crash but I discovered its the CPU that is causing it. This desyncing thing also happens to be a CPU problem. Don't forget that in the AV Player, the replays are broken and desynced. Its the CPU that is causing it. Sometimes when my game crashes, there will be a pop-up saying that a thread from my CPU malfunctioned and deadlocked.

**Comment by A-w-x** [comment]
same issue, i think, see #10727

**Comment by Edenia3** [comment]
This problem can be solved with disabling MSI Afterburner and locking FPS to 30.

**Comment by Back2Life888** [comment]
> This problem can be solved with disabling MSI Afterburner and locking FPS to 30.

Nope, didn't work. I don't even use MSI Afterburner and it still crashes. FPS capping doesn't work also. This is a problem with the CPU timing and coding and stuff.

**Comment by legend800** [comment]
Game is perfectly stable for me. Just tried launch a bunch of times. I haven't seen this issue in years.

Config:
Windows 10 64-bit
GeForce GTX 3070
AMD Ryzen 5900x
32GB RAM

**Comment by AED-001** [comment]
The game is perfectly stable, 60FPS 1080p during gameplay, but it randomly crashes inside of menus like OP stated above. I used to have a problem in previous versions of RPCS3 with being completely unable to play the game at all due to the crashing happening before I could get into a race, but now it only seems to happen after playing the game for a while and exiting a GP back to the menus.

Thankfully, if the game crashes when returning to the menus after winning a GP, the game has already automatically saved so restarting the game will allow you to continue your progress. 

However, it seems that RSGP 14 does not automatically save after the credits sequence. If the game crashes you have to replay RSGP 14 again. Thankfully it's easy and is only 3 races long, unlike the 10-race marathon that precedes it, RSGP 13... :rofl:

Specs:
Xubuntu 21.10
Intel i9 9900K
AMD RX 6800
32GB RAM

Config:
:heavy_check_mark: Enable SPU loop detection
SPU Block Size: Mega
Resolution Scale: 150% (1920x1080)

**Comment by PixelCrusader** [comment]
Ridge Racer 7 is in an obstensibly playable condition, but problems with the game crashing during the intro and ending FMV sequences, and during any attempt to select the special machines for a racing event keep it from being an ideal solution over using the real hardware.

**Comment by Augusto7743** [comment]
RR7 demo run perfectly.
Have you tested using the RR7 3D version update ?

**Comment by juicepoggers** [comment]
This is happening to me both using Vulkan and OpenGL, that's not the issue

**Comment by digant73** [comment]
@juicepoggers still an issue? this issue is also a duplicate of #9555

**Comment by FlexBy420** [comment]
Been playing RR7 few hours at this point, I was unable to reproduce any crash or desync outside of a unrelated HLE bug which is tracked here #18617
